playkit/src/main/java/com/kaltura/playkit/player/VideoCodecSettings.java

```diff
@@ -0,0 +1,72 @@
+package com.kaltura.playkit.player;
+
+import com.kaltura.playkit.PKVideoCodec;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class VideoCodecSettings {
```

Comment:
Remove "video" from all fields and functions

Reply:
done